### PR TITLE
[BALANCE]Buffs the Head of Security's Firearm.

### LINKED ISSUE #8214: HoS Gun Description was written by a Lobotomy Patient
"This is a modern recreation of the antique laser gun. This gun has several unique firemodes, but lacks the ability to recharge over time, its also expensive."

its also expensive

Godammit I'm going to configure git just so this minor conventions error stops bothering me so much

### PR DESCRIPTION
**Pull request by Steelpoint**
This PR buffs the spelling used by the HoS's gun by a factor of ten as requested here: https://github.com/tgstation/-tg-station/issues/8214
Fixes #8214

### CODE REVIEW

**Comment by xxalpha** [comment]
@Steelpoint git on irc m8

File: code/modules/projectiles/guns/energy/nuclear.dm [hunk 1]
```diff
@@ -13,7 +13,7 @@
 	update_icon()
 
 /obj/item/weapon/gun/energy/gun/hos
-	desc = "This is a modern recreation of the antique laser gun. This gun has several unique firemodes, but lacks the ability to recharge over time, its also expensive."
+	desc = "This is an expensive, modern recreation of a antique laser gun. This gun has several unique firemodes, but lacks the ability to recharge over time."
```
Comment (Atskadan) [inline code/modules/projectiles/guns/energy/nuclear.dm]: > a antique laser gun

an

Comment (Steelpoint) [inline code/modules/projectiles/guns/energy/nuclear.dm]: Clearly the gun needs a better buff

**Comment by Dorsisdwarf** [comment]
omg based steelpoint buffin' his job 0/10 demote posthaste.

Good work fixing the dsciption though, it was a bit weid

**Comment by Cheridan** [comment]
TOO OP :-1: :-1: :-1:

**Comment by hornygranny** [comment]
blatant and disgusting HoS favoritism :-1:

**Comment by dannno** [comment]
we all know this will be merged despite the obvious majority against it................................................. no hope......................................

File: code/modules/projectiles/guns/energy/nuclear.dm [hunk 2]
```diff
@@ -13,7 +13,7 @@
 	update_icon()
 
 /obj/item/weapon/gun/energy/gun/hos
-	desc = "This is a modern recreation of the antique laser gun. This gun has several unique firemodes, but lacks the ability to recharge over time, its also expensive."
+	desc = "This is a expensive, modern recreation of a antique laser gun. This gun has several unique firemodes, but lacks the ability to recharge over time."
```
Comment (PKPenguin321) [inline code/modules/projectiles/guns/energy/nuclear.dm]: >a antique

what the fuck steelpoint

**Pull request merged**